.ci/jobs.t/elastic+elasticsearch+periodic+dra-snapshots.yml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: elastic+elasticsearch+%BRANCH%+periodic+dra-snapshot
+    workspace: /dev/shm/elastic+elasticsearch+%BRANCH%+periodic+dra-snapshot
+    display-name: "elastic / elasticsearch # %BRANCH% - DRA snapshot"
+    description: "Publishing Daily Releasable Artifacts (DRAs) of Elasticsearch %BRANCH% snapshots.\n"
+    builders:
+      - inject:
+          properties-file: '.ci/java-versions.properties'
+          properties-content: |
+            JAVA_HOME=$HOME/.java/$ES_BUILD_JAVA
+            RUNTIME_JAVA_HOME=$HOME/.java/$ES_RUNTIME_JAVA
+      - shell: |
+            #!/usr/local/bin/runbld --redirect-stderr
+            DRA_VERSION=`cat build-tools-internal/version.properties \
```

Comment:
I'd just call this `ES_VERSION` since that's what it is and there's noting "dra-specific" about how we version artifacts.